### PR TITLE
[WebProfilerBundle] Replace last "old" icon + delete ICONS_LICENCE.txt

### PR DESCRIPTION
**Pull request by smnandre**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no 
| License       | MIT

Two (related) things in this PR 
* replace two icons used in the Messenger profiler panel with a Tabler-Icon one (following the conventions used since  the [profiler redesign](https://github.com/symfony/symfony/pull/47148) )
* delete the now outdated `ICONS_LICENCE.txt` file (as there is no concerned icons remaining, and there is now a [LICENCE.txt](https://github.com/symfony/symfony/blob/6.4/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Icon/LICENSE.txt) file in the `Resources/views/Icon/` directory)

I targeted the 6.4 branch as there is no BC in code.. but i don't know how to handle a licence change 🤷🏻‍♂️

---

| Previous icons | After this PR | 
| - | - |
| previous-icons | tabler-icon |

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig [hunk 1]
```diff
@@ -131,10 +132,10 @@
                     {% if dispatchCall.exception is defined %}
                         <span class="label status-error">exception</span>
                     {% endif %}
-                    <a class="toggle-button">
-                        <span class="icon icon-close">{{ source('@WebProfiler/images/icon-minus-square.svg') }}</span>
-                        <span class="icon icon-open">{{ source('@WebProfiler/images/icon-plus-square.svg') }}</span>
-                    </a>
+                    <button class="btn btn-link toggle-button" type="button">
+                        <span class="icon icon-close">{{ source('@WebProfiler/Icon/chevron-down.svg') }}</span>
+                        <span class="icon icon-open">{{ source('@WebProfiler/Icon/chevron-down.svg') }}</span>
```
Comment (fabpot) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig:137]: One of them should be chevron-up, right?

**Comment by javiereguiluz** [comment]
Simon, thanks for this improvement. I like it!

About the removal of the previous icon license, I'd say there's no problem.

-----

While reviewing this, I found two other occurrences of `toggle-button` in the `form` panel:

https://github.com/symfony/symfony/blob/d8a4b3c16a08be15aa24e0899bbdcb34c3fef002/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig#L408-L410

https://github.com/symfony/symfony/blob/d8a4b3c16a08be15aa24e0899bbdcb34c3fef002/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig#L500-L502

Would you have some time to try to update those too? If you can't, we can tackle that in another PR. Thanks.

**Comment by smnandre** [comment]
> While reviewing this, I found two other occurrences of `toggle-button` in the `form` panel:

That's not the same icon: it uses a "[Tabler icon but in base64](https://github.com/symfony/symfony/blob/d8a4b3c16a08be15aa24e0899bbdcb34c3fef002/src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig#L100)" ..  would you like to change it for a "chevron" ? 

(i rather ask if you did not see)

**Comment by javiereguiluz** [comment]
@smnandre I didn't realize they were different icons. Sorry.

In any case, we should probably be consistent and use a chevron in all examples related to expand/collapse elements. Another thing; the current icon in form seem to NOT be working because I always see the `[+]`, no matter if it's expanded or collapsed. So, maybe leave it for another PR?

**Comment by smnandre** [comment]
> @smnandre I didn't realize they were different icons. Sorry.

Don't be @javiereguiluz , i missed it entirely so .. :) 

Works for me to leave the rest for another PR..  i'm feeling we may raise other questions / needs.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @smnandre.

**Pull request merged**